step2/README.md

```diff
@@ -1,8 +1,73 @@
-# Step 2
+# Step 2: Get rolling
 
 Objectives:
- - Allow the ball to be controlled by tilting the device
+ - Detect device tilts by the user
+ - Control the ball by tilting the device
 
 ## Steps to get to this point
 
-**To be added**
+### 1: Add a DeviceMotion listener
+
+We need to signal to the device's native API that we want to subscribe to tilt events. The `expo-sensors` module handles all of this: its export `DeviceMotion` has a method `addListener` that bridges to the appropriate Android or iOS system API and starts listening to device tilts and movements.
+
+We want to add this listener when the screen is mounted and rendered - a "side effect" of rendering the screen. The typical way to handle side effects of a render like this in React is with the built-in `useEffect` hook. It takes a function and an array, and calls the function once when the component is rendered, and again if the contents of the array change (but we don't want to call it again here, so we pass an empty array here). If the function returns another function, it calls the returned function when the component is unmounted or removed.
+
+Initially, to see how `DeviceMotion` works, let's add a listener that just logs the details of the device tilt to the console:
+
+```js
+  React.useEffect(() => {
```

Comment:
is DeviceMotion just available, or does there need to be an import?

Reply:
All the imports were added in an earlier step, I think to make the changes " each step simpler. Maybe it makes sense to change that and add them when used since we have text introducing each dep and we'll have the PR diffs to refer to.

Reply:
Maybe something for a future iteration. As long as the imports are already in there then this is fine for next weeks workshop.